src/App/App.js

```diff
@@ -29,11 +55,21 @@ render() {
         return (
         <>
         <Nav/>
-        <main className='movie-description'>
-            {console.log(this.state.movie)}
-        <h1>{this.state.movie.title}</h1>
-        <img className='single-image' src={this.state.movie.poster_path} alt={this.state.movie.title}/>
+        <main className='movie-container'>
+        <article className='image-container'>
```

Comment:
In this area I believe we can destruct the state of the movie to refactor the code to look more cleaner.